serverless/networking/serverless-ossm-jwt.adoc

```diff
@@ -0,0 +1,26 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm-jwt"]
+= Using JSON Web Token authentication with {ProductShortName} and {ServerlessProductName}
+:context: serverless-ossm-jwt
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+// what benefits does this integration provide?
+
+.Prerequisites
+. Install xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[{ServerlessProductName}].
+. Install xref:../../service_mesh/service_mesh_install/preparing-ossm-installation.adoc#preparing-ossm-installation[{ProductName}].
+. Configure xref:../../serverless/networking/serverless-ossm.adoc#serverless-ossm[Using {ProductShortName} with {ServerlessProductName}].
```

Comment:
I think you can remove "Using" from the link name -> Configure [Service Mesh with OpenShift Serverless].